notebook/templates/notebook.html

```diff
@@ -109,6 +109,7 @@
                                 <li id="download_ipynb"><a href="#">{% trans %}Notebook (.ipynb){% endtrans %}</a></li>
                                 <li id="download_script"><a href="#">{% trans %}Script{% endtrans %}</a></li>
                                 <li id="download_html"><a href="#">{% trans %}HTML (.html){% endtrans %}</a></li>
+                                <li id="download_slides"><a href="#">{% trans %}Reveal.js Slides (.html){% endtrans %}</a></li>
```

Comment:
I'd say that 'slides' should be lower case.

Reply:
OK, will make the change.